mmv1/third_party/terraform/acctest/vcr_utils.go

```diff
@@ -14,6 +14,7 @@ import (
 	"path/filepath"
 	"reflect"
 	"regexp"
+	"slices"
```

Comment:
Self review: the `slices` package was added in Go 1.21 - as of this week (1.23 was just released) that's a couple of major versions ago but I though I'd call it out in case it's a problem.